src/schema/basic_classes.yaml

```diff
@@ -444,12 +444,9 @@ slots:
     range: PersonValue
     description: Principal Investigator who led the study and/or generated the dataset.
   was_generated_by:
-    range: WorkflowExecution
+    range: PlannedProcess
```

Comment:
from DM call:
the grouping here is that these two create a "dataobject", e.g. a "DataOutputter" class.

Reply:
DataEmittingProcess?